dydx/dydx.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,5 +1,5 @@
 {
-  "originHash" : "e085067eee7983e4ba42c06fd96e9ea3884445575db37e125bed61bde5eeb3f1",
+  "originHash" : "3cd7346cace16cf660f9c22302c5ca6770335c67902f3fd155011356a9d43929",
```

Comment:
@ruixhuang any chance you know why my package resolved file just keeps updating? It's all over the place

Reply:
Yeah.. it happens to me, I'd just revert it without checking it in.

Reply:
i might [remove](https://forums.swift.org/t/package-resolved-should-go-in-the-gitignore/14699)